### ISSUE TITLE
update file info needs to be seen on live dataset(s)

### ISSUE THREAD

**Issue by only1chunts** [gigascience/gigadb-website#1175]
Initially when I wrote this ticket I thought it was a caching issue, but I have since tried to clear my cache with no effect, I even looked at the dataset in a completely different browser and it still has the old location?! so have upgraded this to a high impact bug instead of low. (I = CIH)

## User story

>As a curator
>I want to be able to update the file location or other file details via the admin pages and see those immediately on the live dataset
>So that the file page is updated immediately without needing to clear the cache

## Acceptance criteria

>Given I navigate to a public dataset page (e.g. https://beta.gigadb.org/dataset/view/id/100216)
> and I notice something that needs updating
>When I update the file location in the admin pages
>Then I can see the new location on the public dataset page without needing to clear the cache.

## Product Backlog Item Ready Checklist

* [ ] Business value is clearly articulated
* [ ] Item is understood enough by the IT team so it can make an informed decision as to whether it can complete this item
* [ ] Dependencies are identified and no external dependencies would block this item from being completed
* [ ] At the time of the scheduled sprint, the IT team has the appropriate composition to complete this item
* [ ] This item is estimated and small enough to comfortably be completed in one sprint
* [ ] Acceptance criteria are clear and testable
* [ ] Performance criteria, if any, are defined and testable
* [ ] The Scrum team understands how to demonstrate this item at the sprint review

## Product Backlog Item Done Checklist

* [ ] Item(s) in increment pass all Acceptance Criteria
* [ ] Code is refactored to best practices and coding standards
* [ ] Documentation is updated as needed
* [ ] Data security has not been compromised (with particular reference to the personal information we hold in GigaDB)
* [ ] No deviation from the team technology stack and software architecture has been introduced
* [ ] The product is in a releasable state (i.e. the increment has not broken anything)

**Comment by rija** [comment]
First, try to reproduce on AWS deployment
If reproduced, look at the invalidation query in the yii config file ( likely protected/config/main.php)

**Comment by kencho51** [comment]
Background work:
1. The file location can be updated in local dev
2. On staging after updating the location to `https://test.com/MESOMICS_data-main.zip`, the `http://ec2-staging.gigadb.link/dataset/102342` cannot show the updated value.
<img width="1083" alt="image" src="https://user-images.githubusercontent.com/64770635/208840509-888f5077-1018-4986-a318-5edc9b458b01.png">
<img width="543" alt="image" src="https://user-images.githubusercontent.com/64770635/208840572-02aca75e-3852-4226-9e97-8c53f80df246.png">
<img width="1284" alt="image" src="https://user-images.githubusercontent.com/64770635/208841072-93daf014-1b2f-4648-b1fa-f14e894a53d6.png">

**Comment by pli888** [comment]
Probably invalidation of cache problem.

**Comment by only1chunts** [comment]
This appears to be an issue on beta server when updating file information using the uploadscripts too, see email from chris subject line "Issue with upload tool on beta" dated May 31, 2023 (pasted below).


I've just uploaded a dataset from an excel file (attached) using the upload script, it appeared to work fine. When I went to the dataset page in the beta server ti displayed most of the dataset, but its only showing 2 of the files?! (see first image below)
Yet when I goto the files admin page it shows that all the file details have been uploaded (see the 2nd image below)

![image](https://github.com/gigascience/gigadb-website/assets/6037145/bd8ed7d1-5a18-4818-86ce-4c71987d24fd)

![image](https://github.com/gigascience/gigadb-website/assets/6037145/9431e9e3-87a0-42db-beaf-9ba009cd85d7)




I think the script may be working, but for some reason the website has cached a previous version of the dataset I uploaded a few days ago (which was removed with the daily updates of the database).
